### DISCUSSION TITLE
Add pycln

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [astral-sh/ruff#14618] [Ideas]
Would it be possible to add [pycln](https://hadialqattan.github.io/pycln/#/) to `ruff`?

**Comment by dhruvmanila** [comment]
I think that should be covered by https://docs.astral.sh/ruff/rules/unused-import/ which also provides an auto-fix to remove the import statement.

**Reply by adam-grant-hendry** [reply]
Even better ❤️. I love chopping away at dependencies from my project one at a time 😃
